package.json

```diff
@@ -1,55 +1,38 @@
 {
-  "author": "Jared Allard <jaredallard@outlook.com>",
   "name": "nexe",
-  "description": "create single executables out of your [node/io].js applications",
+  "description": "Create a single executable out of your Node.js application",
   "license": "MIT",
-  "version": "1.1.2",
+  "version": "2.0.0-beta.1",
   "contributors": [
-    {
-      "name": "Criag Condon",
-      "email": "craig.j.condon@gmail.com",
-      "url": "http://crcn.io/"
-    }
+    "Craig Condon <craig.j.condon@gmail.com> (http://crcn.io)",
+    "Jared Allard <jaredallard@outlook.com>",
+    "Caleb Boyd <caleb.boyd@hotmail.com>"
   ],
   "scripts": {
-    "test": "echo node: $(node -v) && node_modules/.bin/mocha --reporter mocha-circleci-reporter test/test.js"
+    "lint": "eslint index.js ./src/**/*",
+    "test": "npm run lint"
```

Comment:
I totally agree with you! I haven't had a chance to figure out how exactly testing should work. But it is definitely something that will be added before before I will even consider saying its ready.